packages/contracts/contracts/foundry/SphinxUtils.sol

```diff
@@ -983,4 +980,49 @@ contract SphinxUtils is SphinxConstants, StdUtils {
             revert("Sphinx: Invalid MerkleRootStatus. Should never happen.");
         }
     }
+
+    function create2Deploy(bytes memory _initCodeWithArgs) public returns (address) {
```

Comment:
Moved this function and `deploySphinxSystem` below from `SphinxTestUtils` to `SphinxUtils` because these are both required to deploy the system contracts when estimating the Merkle leaf gas fields